honeybee_radiance_command/ra_gif.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+
+"""ra_gif command."""
+
+from .options.ra_gif import Ra_GIFOptions
+from ._command import Command
+import honeybee_radiance_command._typing as typing
+import honeybee_radiance_command._exception as exceptions
+
+
+class Ra_GIF(Command):
+    """ra_gif command."""
+
+    __slots__ = ('_input')
+
+    def __init__(self, options=None, output=None, input=None):
+        Command.__init__(self, output=output)
+        self._input = input
+        if options:
+            self._options = options
+        else:
+            self._options = Ra_GIFOptions()
+
+    @property
+    def options(self):
+        """RaGIF options."""
+        return self._options
+
+    @options.setter
+    def options(self, value):
+        if not value:
+            value = Ra_GIFOptions()
+
+        if not isinstance(value, Ra_GIFOptions):
+            raise ValueError('Expected RaGIFoptions not {}'.format(value))
+
+        self._options = value
+
+    @property
+    def input(self):
+        """Radiance HDR image."""
+        return self._input
+
+    @input.setter
+    def input(self, value):
+        if value[-4:] != '.hdr':
+            raise ValueError(
+                'Radiance generated HDR required. Instead got {}'.format(value))
+        else:
+            self._input = typing.normpath(value)
+
+    def to_radiance(self, stdin_input=False):
+        """Command in Radiance format.
+
+        Args:
+            stdin_input: A boolean that indicates if the input for this command comes
+            from stdin. This is for instance the case when you pipe the input
+            from another command (default: False).
+        """
+        self.validate(stdin_input)
+
+        command_parts = [self.command, self.options.to_radiance()]
+        cmd = ' '.join(command_parts)
+        if not stdin_input and self.input:
+            cmd = ' < '.join((cmd, self.input))
```

Comment:
You don't need `<` for input file based on the documentation. The same for `>` with output. If I remember correctly you can execute the command as:

`ra_gif input.hdr output.gif`

I haven't used this command for years so please double check it yourself.

Reply:
Your memory serves you well. https://github.com/ladybug-tools/honeybee-legacy/blob/6d5098d13f80a287cbac75385f8876aedc5d5f94/src/Honeybee_Convert%20HDR%20to%20TIF.py#L121

When I see any command, set for example [pcond](https://floyd.lbl.gov/radiance/man_html/pcond.1.html), I don't see "<" for input or ">" for output. The same is the case with rtrace. 

I believed that this is done to use the [input redirection](https://www.javatpoint.com/linux-input-redirection) in Linux. Am I wrong to believe this? If I am then this needs to change in rtrace as well.

Reply:
Some Radiance commands like rtrace accepts stdin for input and print out everything to stdout. That's why we use < to read in the input values and > to redirect the outputs to a file instead of stdout.

It's not the case for all the commands like ra_gif or pcond. In these commands inputs and outputs are command arguments. That's hwy you don't need > or <.